include/exadg/postprocessor/pointwise_output_generator_base.cpp

```diff
@@ -127,6 +126,12 @@ PointwiseOutputGeneratorBase<dim, Number>::setup_base(
 #ifdef DEAL_II_WITH_HDF5
   pointwise_output_data = pointwise_output_data_in;
 
+  time_control.reinit(pointwise_output_data_in.write_output,
+                      pointwise_output_data_in.start_time,
+                      pointwise_output_data_in.end_time,
+                      pointwise_output_data_in.interval_time,
+                      0);
```

Comment:
why 0?

Reply:
this was just to ensure everything compiles before refracturing